packages/angular-test/src/text.ts

```diff
@@ -10,11 +10,13 @@ export const textTest = <C extends JsonFormsControl>(
     imports: any[],
     providers: any[],
     componentUT: Type<C>,
-    errorTestInformation: ErrorTestInformation) => () => {
+    instance: string,
+    errorTestInformation: ErrorTestInformation,
+    elementToUse: (element: DebugElement) => any) => () => {
```

Comment:
Minor: Use consistent naming for parameter `elementToUse` and arguments `toSelect`, maybe `nativeElementSelector` or similar.

Reply:
I would add this to #1100 and do a full clean up